app/assets/templates/header_tpl.jst.hbs

```diff
@@ -94,6 +94,7 @@
                 {{#if current_user.admin}}
                   <li><a href="/admins/dashboard">{{t "header.admin"}}</a></li>
                 {{/if}}
+                <li class="visible-sm visible-xs"><a href="/mobile/toggle">{{t "header.toggle_mobile"}}</a></li>
```

Comment:
I'd like other opinions here, but I would go for a use of the already existing translation. This would save work for translators.